### PR TITLE
[#16077] Add community illustrations

### LINKED ISSUE #16077: Communities Home Page - Add illustrations and adjust copy
The design team have revisited the content for the empty states of the Communities Home Page-

see the designs here:
https://www.figma.com/file/NAdciBFYSBXmr9wpT47jY1/Illustrations?type=design&node-id=167-65517&t=NP7OaDqj6j3JmZJc-0

Screenshot 2023-05-31 at 11 09 00

Implement the copy for all of the sections in the "Communities"  ->
No joined, No Pending, No Opened, No Contacts

### PR DESCRIPTION
**Pull request by ulisesmac**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

fixes #16077

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)

Adds the communtiy illustrations shown when the content  in the tabs is empty according to [designs](https://www.figma.com/file/NAdciBFYSBXmr9wpT47jY1/Illustrations?type=design&node-id=167-65517&t=F7DlRfPtbFAPouaV-4)
image

image

#### Platforms

- Android
- iOS

### Steps to test

- Open Status
- Navigate to communities tab
- Make sure you have an empty state (no communities joined, pending and opened) to show the new images

status: ready

### CODE REVIEW

File: src/status_im2/contexts/communities/home/view.cljs [hunk 1]
```diff
@@ -56,26 +75,25 @@
           {:keys [joined pending opened]} (rf/sub [:communities/grouped-by-status])
           {:keys [key-uid]}               (rf/sub [:multiaccount])
           account                         (rf/sub [:profile/multiaccount])
-          profile-color                   (:color (rf/sub [:onboarding-2/profile]))
-          customization-color             (if profile-color
-                                            profile-color
-                                            (rf/sub [:profile/customization-color key-uid]))
+          customization-color             (or (:color (rf/sub [:onboarding-2/profile]))
+                                              (rf/sub [:profile/customization-color key-uid]))
```
Comment (ulisesmac) [inline src/status_im2/contexts/communities/home/view.cljs]: Just a small refactor

**Review by J-Son89 — APPROVED** [review]

File: src/status_im2/contexts/communities/home/view.cljs [hunk 2]
```diff
@@ -34,20 +35,38 @@
...
+    :pending
+    {:title       (i18n/label :t/no-pending-communities)
+     :description (i18n/label :t/no-pending-communities-description)
+     :image       (resources/get-image (theme/theme-value :no-pending-communities-light
+                                                          :no-pending-communities-dark))}
+    :opened
+    {:title       (i18n/label :t/no-opened-communities)
+     :description (i18n/label :t/no-opened-communities-description)
+     :image       (resources/get-image (theme/theme-value :no-opened-communities-light
+                                                          :no-opened-communities-dark))}))
```
Comment (ilmotta) [inline src/status_im2/contexts/communities/home/view.cljs]: `case` needs a default option, otherwise it throws an exception when no clause matches.

**Review by ilmotta — APPROVED** [review]

File: src/status_im2/contexts/communities/home/view.cljs (hunk 2 above)
Comment (ulisesmac) [reply src/status_im2/contexts/communities/home/view.cljs]: Just added `nil` since we currently have a fixed set of fixed tabs, thanks for pointing :+1:

**Comment by J-Son89** [comment]
Can you please verify these visual changes using iPhone 11 Pro (or iPhone 12 Pro) and Figma? Let me know if you need any help with this 👍

**Comment by qoqobolo** [comment]
@ulisesmac could you check the integration tests here, please?

**Comment by qoqobolo** [comment]
### ISSUE 1: The image remains dark if change the device theme from light to dark (only for the first time and only on Pixel 7a :D)

1. Install the app 
2. Open it and change the device theme

It is fixed after reopening the app.


https://github.com/status-im/status-mobile/assets/67952253/1cb8a563-ce83-45ed-93c8-d7583dee8b39

### ISSUE 2: The position of the illustration should be adjusted to fit different screen sizes

iPhone SE

Screenshot 2023-06-05 at 12 26 32

Huawei P40 lite

Screenshot 2023-06-05 at 12 26 50

Pixel 7a

Screenshot 2023-06-05 at 12 27 31

**Comment by ulisesmac** [comment]
Hey @qoqobolo 

About the issue 2, yes, I already talked to Mario about that, I will refactor the code a little and I will let you know when it's done!
Thanks!!

File: src/status_im2/contexts/communities/home/view.cljs [hunk 3]
```diff
@@ -34,73 +35,92 @@
...
+       :image               image
+       :title               title
+       :description         description}]]))
 
 (defn home
   []
-  (fn []
-    (let [selected-tab                    (or (rf/sub [:communities/selected-tab]) :joined)
-          {:keys [joined pending opened]} (rf/sub [:communities/grouped-by-status])
-          {:keys [key-uid]}               (rf/sub [:multiaccount])
```
Comment (ulisesmac) [inline src/status_im2/contexts/communities/home/view.cljs:57]: It was a form-2 component but it was not required, so changed it to form-1

File: src/status_im2/contexts/communities/home/view.cljs [hunk 4]
```diff
@@ -34,73 +35,92 @@
...
-                                            :pending pending
-                                            :opened  opened)
-          top                             (safe-area/get-top)]
-      [:<>
-       (if (empty? selected-items)
-         [empty-state]
-         [rn/flat-list
-          {:key-fn                            :id
-           :content-inset-adjustment-behavior :never
-           :header                            [rn/view (style/header-height top)]
```
Comment (ulisesmac) [inline src/status_im2/contexts/communities/home/view.cljs:74]: This `view` component is receiving styles outside a `style` key, I added the key

**Comment by ulisesmac** [comment]
> ### ISSUE 1: The image remains dark if change the device theme from light to dark (only for the first time and only on Pixel 7a :D)
> 
>     1. Install the app
> 
>     2. Open it and change the device theme
> 
> 
> It is fixed after reopening the app.
> video_2023-06-05_12-23-11.mp4
> ### ISSUE 2: The position of the illustration should be adjusted to fit different screen sizes
> 
> iPhone SE
> Screenshot 2023-06-05 at 12 26 32
> 
> Huawei P40 lite
> Screenshot 2023-06-05 at 12 26 50
> 
> Pixel 7a
> Screenshot 2023-06-05 at 12 27 31

@qoqobolo 

Seems like an edge case, the best IMO would be open an issue and investigatre it later. Currently, I don't know what's happening and why only on that phone, I was not able to replicate it on Pixel 6 emulator.

**Comment by ulisesmac** [comment]
> Hey @qoqobolo
> 
> About the issue 2, yes, I already talked to Mario about that, I will refactor the code a little and I will let you know when it's done! Thanks!!

@qoqobolo Fixed! Please test it again, thanks

**Comment by qoqobolo** [comment]
Looks really nice, thanks @ulisesmac!
PR can be merged.

**Pull request merged**